app/views/deliveries/index.html.erb

```diff
@@ -0,0 +1,107 @@
+<head>
+  <style>
+  
+  .waiting_on{
+    background-color:#fafad2;
+  }
+
+  .received{
+    background-color:#90ee8f;
+  }
+  </style>
+</head>
+
+<body>
+
+<div>
+  <div>
+    <h1>
+      Delivery Tracker
+    </h1>
+
+    <p>Know if something gets lost in the mail.</p>
+  </div>
+</div>
+
+<hr>
+
+<div>
+  <div>
+    <h2>
+      Expecting a package?
+    </h2>
+
+    <form action="/insert_delivery" method="post">
+    <input type="hidden" name="query_user_id" value="<%=current_user.id%>">
```

Comment:
nice way of getting the current users id :). you will eventually learn a better way that is less prone to security risks